settings.py

```diff
@@ -716,12 +716,6 @@ def get_locales():
         ),
         'output_filename': 'build/styles/fellowship.css',
     },
-    'mdn10': {
-        'source_filenames': (
-            'css/mdn10.css',
-        ),
-        'output_filename': 'build/styles/mdn10.css',
-    },
```

Comment:
It was being included in the zone CSS as an @import but I copy and pasted it into the zone instead so we can remove it from here. I thought it was a risk to keep it in the code base without it being linked to from anywhere in the code base.